src/main/webapp/style.css

```diff
@@ -260,3 +260,85 @@ h1 {
   border-radius: 5px;
   padding: 5px 10px 5px 10px;
 }
+
+/* individual event display */
+.event-display-container {
+  display: flex;
+  flex-flow: row nowrap;
+  height: calc(100% - 100px);
+  width: 100%;
+  position: fixed;
+}
+
+.event-left-details {
+  height: 100%;
+  flex-grow: 1;
+  overflow-y: scroll;
+  display: flex;
+  flex-flow: column nowrap;
+}
+
+.event-display-image {
+  height: 400px;
+  width: 100%;
+  background-color: #93c47d;
+  flex: 0 0 auto;
+}
+
+.event-left-text {
+  padding: 0 15px 30px 15px;
+}
+
+.event-display-header {
+  display: flex;    
+  flex-flow: row nowrap;
+  align-items: center;
+  justify-content: space-between;
+  padding: 30px 0 15px 0;
+}
+
+.event-display-title {
+  font-weight: bold;
+  font-size: 25px;
+}
+
+.event-display-description {
+  margin-bottom: 30px;
+}
+
+.event-right-details {
+  background-color: #eee;
+  font-size: 18px;
+  margin: 0;
+  width: 30%;
+  flex: 0 0 auto;
+  min-width: 350px;
+  max-width: 500px;
+  height: 100%;
+  display: flex;
+  flex-flow: column nowrap;
+  align-items: center;
+  justify-content: center;
+}
+
+.event-time-location {
+  margin: 15px 15px 50px 15px;
+}
+
+.save-event, .go-back {
+  width: 200px;
+  height: 75px;
+  margin: 15px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  cursor: pointer;
+}
```

Comment:
I believe CSS declarations are supposed to be in alphabetical order according to the style guides

Reply:
Fixed!